jbpm-wb-kie-server/jbpm-wb-kie-server-backend/src/main/java/org/jbpm/workbench/ks/integration/KieServerDataSetProvider.java

```diff
@@ -414,11 +414,25 @@ protected void handleDataSetGroup(final DataSetDef def,
     protected ColumnType getGroupFunctionColumnType(final DataSetDef def,
                                                     final ColumnGroup columnGroup,
                                                     final GroupFunction groupFunction) {
-        ColumnType type = def.getColumnById(groupFunction.getColumnId()).getColumnType();
+        
+        DataColumnDef column = def.getColumnById(groupFunction.getColumnId());
+        if (column == null) {
+            column =  def.getColumnById(groupFunction.getSourceId());
```

Comment:
Hello @caponetto 

I snicked this null checking just to avoid breaking the UI - an annoying issue when using a displayer was that the column id was a generated named, resulting in a null result - in such cases we should use the source id.

However, this code should be improved in a different JIRA and completely rewritten!